### DISCUSSION TITLE
[QUESTION]: Add to existing server

### DISCUSSION THREAD

**Discussion by snakeeater98** [AnOpenSauceDev/FastRandom#52] [Q&A] [answered]
### Does an issue already cover this?

- [X] I have searched for any existing issues, and didn't find any.

### Issue description.

Since you say it alters the world gen a bit, can i add this to a all ready existing vanilla generated world without strange cutoffs when the new world gen kicks in ?

**Comment by AnOpenSauceDev** [comment, marked as answer]
if you use 3.0.0+ there will be very little difference. The worst thing happening might be some caves closing off around chunk borders, and possibly a tree of two being cut in half if between some ungenerated chunk borders. After those border chunks generate, it will be seamless from then on.

**Comment by snakeeater98** [comment]
Tanks for the answer, i assumed these where the appearing effects as well. Would be interesting to do some testing and confirm it.
Maybe in the future ill do some testing.
